types/relay-runtime/lib/store/RelayStoreTypes.d.ts

```diff
@@ -416,88 +416,143 @@ interface OperationDescriptor {
     readonly root: NormalizationSelector;
 }
 
-type LogEvent =
+export type LogEvent =
     | Readonly<{
-          name: 'queryresource.fetch';
-          /**
-           * ID of this query resource request and will be the same if there is an associated queryresource.retain event.
-           */
-          resourceID: number;
-          operation: OperationDescriptor;
-          profilerContext: any;
-          fetchPolicy: FetchPolicy;
-          renderPolicy: RenderPolicy;
-          queryAvailability: OperationAvailability;
-          shouldFetch: boolean;
-      }>
+        name: 'suspense.fragment',
+        data: any,
+        fragment: ReaderFragment,
+        isRelayHooks: boolean,
+        isMissingData: boolean,
+        isPromiseCached: boolean,
+        pendingOperations: ReadonlyArray<RequestDescriptor>,
+    }>
     | Readonly<{
-          name: 'queryresource.retain';
-          resourceID: number;
-          // value from ProfilerContext
-          profilerContext: any;
-      }>
+        name: 'suspense.query',
+        fetchPolicy: string,
+        isPromiseCached: boolean,
+        operation: OperationDescriptor,
+        queryAvailability?: OperationAvailability | undefined,
+        renderPolicy: RenderPolicy,
+    }>
+    | Readonly<{
+        name: 'queryresource.fetch';
+        /**
+         * ID of this query resource request and will be the same if there is an associated queryresource.retain event.
+         */
+        resourceID: number;
+        operation: OperationDescriptor;
+        profilerContext: any;
+        fetchPolicy: FetchPolicy;
```

Comment:
This is defined in referenced Flow code as `string`:
https://github.com/facebook/relay/blob/e938473ba1c674115c8c483d92a5e2efcdd7f000/packages/relay-runtime/store/RelayStoreTypes.js#L489

Not sure if our `FetchPolicy` is a better type, but I'm just pointing out it's different, in case we should match

Reply:
nah i think we can type here, `FetchPolicy` is a string as its base type, we just type narrow to what that "can be".